src/zod/user-login.schema.ts

```diff
@@ -1,7 +1,7 @@
 import { z, type ZodType } from 'zod';
-import { type LoginFormData } from '#/modules/auth/types';
+import { type AuthFormData } from '#/modules/auth/types';
 
-export const UserLoginSchema: ZodType<LoginFormData> = z.object({
+export const UserLoginSchema: ZodType<Pick<AuthFormData, 'email' | 'password'>> = z.object({
   email: z.string().email({ message: 'Please enter a valid email (e.g. john@stayhub.com)' }),
   password: z.string().min(8, { message: 'Entered password is too short' }).max(64, { message: 'Entered password is too long' }),
```

Comment:
Let's don't check password length in login

Reply:
I think it's better to leave it. I tried removing it and backend returned some messy error messages that need to be processed before being shown. This approach will require much more code than there is now

Reply:
> I think it's better to leave it. I tried removing it and backend returned some messy error messages that need to be processed before being shown. This approach will require much more code than there is now

Then password length validation (for login) should be removed from the backend as well. It's not a good practice.

Reply:
fixed